### ISSUE TITLE
Error: line 84. character attribute does not exist in module 'shared'.

### ISSUE THREAD

**Issue by cochese9000** [theubie/complex_memory#17]
Looks like Oob changed their modules\shared.py, if you add this line to the bottom, it works:

```
character = args.character
```

but it might not work when chars change... havent' tested yet.

**Comment by cochese9000** [comment]
Nope, it didn't work.  It just loaded and the chars plugin isn't working.  I think this plugin needs to get the char a different way.  Maybe changing shared.character to shared.args.character or something?

**Comment by weavertron5000** [comment]
FYI I was able to fix this by replacing all references of `shared.character` in complex_memory's script.py to this: shared.settings['character']
